### PR TITLE
Add avatar's provider size props

### PR DESCRIPTION
**Pull request by ThomasPalamara**
## Proposed changes

Simple changes adding a props giving the option of setting the size for the provider icon in the avatar component. Needed for this ticket: https://lightful.atlassian.net/browse/PLATFORM-713

## Types of changes

What types of changes does your code introduce to the project?

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

Please use the following checklist before contributing to this repository.

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have implemented error checking and considered data not being available or loading
- [x] I have added necessary documentation (if appropriate)
- [x] I have implemented correct type checking
- [x] I have tested my changes on a mobile layout
- [x] I have double checked with the team that I haven't introduced duplicates in functionalities (components, services, database changes, etc...)
- [x] If required, designers have signed off my changes
- [ ] I have communicated the changes to the QA team and planned testing
- [ ] I have implemented e2e tests for the new functionality

### CODE REVIEW

File: dist/css/ray.css [hunk 1]
```diff
@@ -6963,19 +6971,6 @@ button.bg-linkedin:focus {
...
-    right: -30%;
-    bottom: -30%; }
-  .avatar-provider-md {
-    width: 60%;
-    height: 60%;
-    right: -10%;
-    bottom: -10%; }
-  .avatar-provider-sm {
-    width: 40%;
-    height: 40%; }
```
Comment (ThomasPalamara) [inline dist/css/ray.css]: I don't understand why you removed the positions? It solves our problem but the render is not right anymore
image

Comment (josephrace) [reply dist/css/ray.css]: I wanted to simplify it and then ask a designer exactly what they're looking for.  Not finished with it...

Comment (ThomasPalamara) [reply dist/css/ray.css]: Ok sorry. Danielle gave me these position values. You should ask her if you want.

Comment (josephrace) [reply dist/css/ray.css]: Ah ok I didn't realise.  I'll try to make it work with these then.

**Comment by josephrace** [comment]
Ok I've changed it a little bit.  I added another div inside to add the padding - I saw issues with adding padding straight to the outer element when the avatar was used on the post card so this seems more robust.

When lined up next to each other (like in Storybook) the avatar sizes vary slightly but I don't think we will ever display them like this in a real world situation.

**Comment by ThomasPalamara** [comment]
Shouldn't we increase the size of the avatar in the postcard component then? It seems too small now.

**Pull request merged**